### PR TITLE
Added font-awesome icons and ListTable component styling changes

### PR DESCRIPTION
**Pull request by xantorres**
Some styling changes to make tables look 
https://zpl.io/2GGkJR9
https://zpl.io/aBRkJdA

I also had to add a reference to the font awesome icons, as it turns out we didn't have them in the project.

image

### CODE REVIEW

File: src/app/static/index.html [hunk 1]
```diff
@@ -5,6 +5,7 @@
     <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
     <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700,900">
     <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/latest/css/font-awesome.min.css">
```
Comment (brennancheung) [inline src/app/static/index.html]: The asset hosted in S3 already has font awesome.

File: src/stories/preview-head.html [hunk 2]
```diff
@@ -1,2 +1,3 @@
 <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500">
 <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/latest/css/font-awesome.min.css">
```
Comment (brennancheung) [inline src/stories/preview-head.html]: Use the one from S3.

**Comment by brennancheung** [comment]
The title `Prometheus Instances` should not be in the table itself, it shows above outside the table component.

Also the edit/delete icons have text underneath them.

File: src/app/core/components/listTable/ListTableColumnSelector.js [hunk 3]
```diff
@@ -1,25 +1,30 @@
 import React from 'react'
 import PropTypes from 'prop-types'
-import { IconButton, Popover, Tooltip } from '@material-ui/core'
+import { Popover, Tooltip } from '@material-ui/core'
 import { withStyles } from '@material-ui/styles'
-import ViewColumnIcon from '@material-ui/icons/ViewColumn'
 import grey from '@material-ui/core/colors/grey'
 import { compose } from 'ramda'
 import ListTableColumnPopover from 'core/components/listTable/ListTableColumnPopover'
+import * as classnames from 'classnames'
```
[the remark this answers is not in the document]
Comment (xantorres) [reply src/app/core/components/listTable/ListTableColumnSelector.js]: yes, I always do that, I missed it in this case

**Pull request merged**